### PR TITLE
Support basepath from servers

### PR DESCRIPTION
**Pull request by leolplex**
Hello @StefH ,

This change support the basepath definition in v2 and servers in  v3. The idea is when the file has a relative or absolute path this is attached at the path.

Thanks.

### CODE REVIEW

File: src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs [hunk 1]
```diff
@@ -230,6 +230,23 @@ private string MapPathWithParameters(string path, IEnumerable<OpenApiParameter>
             return newPath;
         }
 
+        private string MapBasePath(IList<OpenApiServer> servers)
+        {
+            if (servers == null || servers.Count == 0)
+            {
+                return null;
```
Comment (StefH) [inline src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs:237]: Better to return `string.Empty` here?

File: src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs [hunk 2]
```diff
@@ -230,6 +230,23 @@ private string MapPathWithParameters(string path, IEnumerable<OpenApiParameter>
...
+            }
+
+            OpenApiServer server = servers.First();
+            Uri uriResult;
+            if (Uri.TryCreate(server.Url, UriKind.RelativeOrAbsolute, out uriResult))
+            {
+
```
Comment (StefH) [inline src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs:244]: remove empty line

File: src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs [hunk 3]
```diff
@@ -230,6 +230,23 @@ private string MapPathWithParameters(string path, IEnumerable<OpenApiParameter>
...
+                return uriResult.IsAbsoluteUri ? uriResult.AbsolutePath : uriResult.ToString();
+            }
+            return null;
```
Comment (StefH) [inline src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs:247]: Is this correct?

**Review by StefH — CHANGES_REQUESTED** [review]

File: src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs (hunk 1 above)
Comment (leolplex) [reply src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs]: Done!

File: src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs (hunk 2 above)
Comment (leolplex) [reply src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs]: Done!

File: src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs (hunk 3 above)
Comment (leolplex) [reply src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs]: Yes, if the Uril.TryCreate is false then a string.Empty should be returned. Comments?

**Review by StefH — APPROVED** [review]

**Pull request merged**